src/include/common/numeric_utils.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <type_traits>
+
+#include "common/types/int128_t.h"
+#include "common/types/types.h"
+#include <bit>
+#include <concepts>
+
+namespace kuzu {
+namespace common {
+namespace NumericUtils {
```

Comment:
Instead of nesting the namespace `NumericUtils` here, Is it possible to add a utility struct here. Also, for namespace, we always use lower case characters.

Reply:
Updated namespace name, I'd prefer keeping it as a namespace over a struct as there is no state being stored